Ginger/GingerCoreCommon/ReporterLib/UserMsgsPool.cs

```diff
@@ -310,6 +310,7 @@ public static void LoadUserMsgsPool()
             #endregion Environment Messages
 
             #region Agents/Drivers Messages
+            Reporter.UserMsgsPool.Add(eUserMsgKey.MobileDriverNotConnected, new UserMsg(eUserMsgType.INFO, "Mobile Driver Error", "Mobile driver could not connet to mobile device, please try again.", eUserMsgOption.OK, eUserMsgSelection.None));
```

Comment:
users don't know what is Mobile Driver, set the error to "The connection to the mobile device has failed, please try again."